src/private.rs

```diff
@@ -1,35 +1,24 @@
-use std::fmt::{Debug, Display};
+use std::fmt::{Debug, Display, Write};
 
-#[repr(transparent)]
-pub struct DisplayDebug(String);
-impl std::fmt::Debug for DisplayDebug {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_str(self.0.as_str())
-    }
-}
-impl AsRef<str> for DisplayDebug {
-    #[inline(always)]
-    fn as_ref(&self) -> &str {
-        self.0.as_str()
-    }
-}
-
-pub struct RedactFlags {
+pub enum RedactSpecialization {
     /// Whether the type we're redacting is an Option<T> or not. Poor man's specialization! This is detected
     /// by the proc macro reading the path to the type, so it's not perfect.
     ///
     /// This could be improved & rid of in a number of different ways in the future:
     ///
-    /// * Once specialization is stabilized, we can use a trait to override redacting behaviour for some types,
+    /// * Once specialization is stabilized, we can use a trait to override redacting behavior for some types,
```

Comment:
I can't believe a british dev would do this. Did it hurt?

Reply:
💀